### ISSUE TITLE
Amazon/Netflix logged in, but it's not showing up

### ISSUE THREAD

**Issue by jamesmgg** [RobinLinus/socialmedia-leak#8]
Maybe they fixed the issue, but Amazon/Netflix do not show up as logged in, even though I am.

**Comment by blahRus** [comment]
ditto

**Comment by CDillinger** [comment]
Amazon is showing correctly for me, but it looks like Netflix now just redirects to the main browse page regardless of the redirect in the URL.

**Comment by CDillinger** [comment]
Are you guys logged into Amazon.com specifically, rather than Amazon's site for another language/region? I just logged into Amazon.de, but Amazon.com still showed me logged out.

**Comment by RobinLinus** [comment]
Yep this redirect works only for amazon.com. Idk if it's worth adding other amazon domains...
Thanks for the update on Netflix!

**Comment by CDillinger** [comment]
It looks like Amazon has 14 regional sites... Sounds like overkill to do them all:

Australia - amazon.com.au
Brazil - amazon.com.br
Canada - amazon.ca
China - amazon.cn
France - amazon.fr
Germany - amazon.de
India - amazon.in
Italy - amazon.it
Japan - amazon.co.jp
Mexico - amazon.com.mx
Netherlands - amazon.nl
Spain - amazon.es
United Kingdom - amazon.co.uk
United States - amazon.com

I'm surprised they don't have the logged in account persist across their different regional sites like Google does.
